src/features/page_view_event/aggregate/index.js

```diff
@@ -64,7 +64,7 @@ export class Aggregate extends AggregateBase {
 
     if (globalScope.performance && typeof (globalScope.performance.timing) !== 'undefined') {
       var navTimingApiData = ({
-        timing: addPT(globalScope.performance.timing, {}),
+        timing: addPT(getRuntime(this.agentIdentifier).offset, globalScope.performance.timing, {}),
```

Comment:
Very minor: there's a `agentRuntime` var above that's unused rn.

Reply:
fixed, ty